scripts/deploy.js

```diff
@@ -796,6 +834,18 @@ async function buildUpgradeActionPlan(plan, config) {
   await deployUpgrade(plan, config);
 }
 
+async function buildTransferActionPlan(plan, config) {
+  dotenv.config();
+
+  await getPrivateKey(config);
+  await getRpcUrl(config);
+  await getProvider(config);
+  await getWallet(config);
+
+  await transferIdentityManagerOwnership(config);
```

Comment:
This call should only perform the actions, not ask for config. This is part of the modular way this script is built.